openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1363,6 +1363,9 @@ This specification defines the following Credential Issuer Metadata parameters:
           * `alt_text`: OPTIONAL. String value of the alternative text for the logo image.
       * `description`: OPTIONAL. String value of a description of the Credential.
       * `background_color`: OPTIONAL. String value of a background color of the Credential represented as numerical color values defined in CSS Color Module Level 37 [@!CSS-Color].
+      * `background_image`: OPTIONAL. Object with information about the background image of the Credential. The following non-exhaustive set of parameters MAY be included:
+          * `uri`: REQUIRED. String value that contains a URI where the Wallet can obtain the background image of the Credential from the Credential Issuer. The Wallet needs to determine the scheme, since the URI value could use the `https:` scheme, the `data:` scheme, etc.
+          * `alt_text`: OPTIONAL. String value of the alternative text for the background image.
```

Comment:
A background image wouldn't normally have alt text. We already have alt text for the credential itself, I'm not sure alt text for background image is ever useful. (I'm open to examples of where it might be useful!)

Reply:
ah good point

Reply:
```suggestion
```